File: app/src/main/java/tech/pauly/findapet/data/FilterDatabase.java
package tech.pauly.findapet.data;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;
import android.arch.persistence.room.TypeConverters;

import tech.pauly.findapet.data.models.Converters;
import tech.pauly.findapet.data.models.Filter;

@Database(entities = {Filter.class}, version = 5)
@TypeConverters(Converters.class)
public abstract class FilterDatabase extends RoomDatabase {
    public abstract FilterDao filterDao();
}

File: app/src/main/java/tech/pauly/findapet/data/FilterDatabase.kt
package tech.pauly.findapet.data

import androidx.room.Database
import androidx.room.RoomDatabase
import androidx.room.TypeConverters

import tech.pauly.findapet.data.models.Converters
import tech.pauly.findapet.data.models.Filter

@Database(entities = [(Filter::class)], version = 6)
@TypeConverters(Converters::class)
abstract class FilterDatabase : RoomDatabase() {
    abstract fun filterDao(): FilterDao
}
